`make local` fails with `resourceKind` required error

```
kubectl delete -f deploy/crds/apps_v1alpha1_servicebindingrequest_cr.yaml
Error from server (NotFound): error when deleting "deploy/crds/apps_v1alpha1_servicebindingrequest_cr.yaml": servicebindingrequests.apps.openshift.io "example-servicebindingrequest" not found
make: [Makefile:310: deploy-clean] Error 1 (ignored)
kubectl delete -f deploy/crds/apps_v1alpha1_servicebindingrequest_crd.yaml
customresourcedefinition.apiextensions.k8s.io "servicebindingrequests.apps.openshift.io" deleted
kubectl delete -f deploy/operator.yaml
Error from server (NotFound): error when deleting "deploy/operator.yaml": deployments.apps "service-binding-operator" not found
make: [Makefile:312: deploy-clean] Error 1 (ignored)
kubectl delete -f deploy/role_binding.yaml
rolebinding.rbac.authorization.k8s.io "service-binding-operator" deleted
kubectl delete -f deploy/role.yaml
role.rbac.authorization.k8s.io "service-binding-operator" deleted
kubectl delete -f deploy/service_account.yaml
serviceaccount "service-binding-operator" deleted
kubectl create -f deploy/service_account.yaml
serviceaccount/service-binding-operator created
kubectl create -f deploy/role.yaml
role.rbac.authorization.k8s.io/service-binding-operator created
kubectl create -f deploy/role_binding.yaml
rolebinding.rbac.authorization.k8s.io/service-binding-operator created
kubectl create -f deploy/crds/apps_v1alpha1_servicebindingrequest_crd.yaml
customresourcedefinition.apiextensions.k8s.io/servicebindingrequests.apps.openshift.io created
kubectl apply -f deploy/crds/apps_v1alpha1_servicebindingrequest_cr.yaml
The ServiceBindingRequest "example-servicebindingrequest" is invalid: []: Invalid value: map[string]interface {}{"apiVersion":"apps.openshift.io/v1alpha1", "kind":"ServiceBindingRequest", "metadata":map[string]interface {}{"annotations":map[string]interface {}{"kubectl.kubernetes.io/last-applied-configuration":"{\"apiVersion\":\"apps.openshift.io/v1alpha1\",\"kind\":\"ServiceBindingRequest\",\"metadata\":{\"annotations\":{},\"name\":\"example-servicebindingrequest\",\"namespace\":\"default\"},\"spec\":{\"applicationSelector\":{\"matchLabels\":{\"connects-to\":\"postgres\",\"environment\":\"production\"},\"resourceKind\":\"Deployment\"},\"backingServiceSelector\":{\"resourceName\":\"databases.example.org\",\"resourceRef\":\"pg-instance\",\"resourceVersion\":\"v1alpha1\"},\"mountPathPrefix\":\"/var/credentials\"}}\n"}, "creationTimestamp":"2019-07-30T17:45:55Z", "generation":1, "name":"example-servicebindingrequest", "namespace":"default", "uid":"e1859d07-b2f1-11e9-96c1-080027d8ec88"}, "spec":map[string]interface {}{"applicationSelector":map[string]interface {}{"matchLabels":map[string]interface {}{"connects-to":"postgres", "environment":"production"}, "resourceKind":"Deployment"}, "backingServiceSelector":map[string]interface {}{"resourceName":"databases.example.org", "resourceRef":"pg-instance", "resourceVersion":"v1alpha1"}, "mountPathPrefix":"/var/credentials"}}: validation failure list:
spec.backingServiceSelector.resourceKind in body is required
spec.envVarPrefix in body is required
make: *** [Makefile:305: deploy-cr] Error 1
```

--- a/deploy/crds/apps_v1alpha1_servicebindingrequest_cr.yaml
+++ b/deploy/crds/apps_v1alpha1_servicebindingrequest_cr.yaml
@@ -7,7 +7,7 @@ spec:
   mountPathPrefix: "/var/credentials"
   backingServiceSelector:
     resourceRef: pg-instance
-    resourceName: databases.example.org
+    resourceKind: databases.postgresql.example.dev
     resourceVersion: v1alpha1
   applicationSelector:
     matchLabels:

